connector/connect/README.md

```diff
@@ -24,7 +24,30 @@ or
 ```bash
 ./build/sbt -Phive clean package
 ```
-   
+
+### Build with user-defined `protoc` and `protoc-gen-grpc-java`
```

Comment:
I would add an example in which we need this custom compilation. For example, CentOS 5 and 6.

Reply:
done